Financial-Times/n-messaging-client
# UG-885 Implement config, styles and behaviour for Academia sharing message

Adds config, styles, and client-side behaviour for the Academia sharing message. The message is lazy, and retrieves it's HTML structure from next-messaging-guru (see https://github.com/Financial-Times/next-messaging-guru/pull/178).

In order to test:
1. Set the `messageSlotBottom` flag to the variant `AcademiaCTA`
2. Set the `academicCtaTest` flag to `variant`
3. Checkout the associated [next-messaging-guru PR](https://github.com/Financial-Times/next-messaging-guru/pull/178) and run using `make run`
4. Checkout this PR and run the app using `make demo-with-guru`
5. Go to https://local.ft.com:5005/
6. The message should appear as per [the designs](https://www.figma.com/file/09mlE9M0XVG5fIZy99tls2/(Final)-Professor-Engagement)
